provider/ghasedak/ghasedak.go

```diff
@@ -0,0 +1,64 @@
+package ghasedak
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/messagebird/sachet"
+)
+
+// Retrieving required data from 'ghasedak' sections of config.yaml
+type Config struct {
+	APIToken     string   `yaml:"api_token"`
+	PhoneNumbers []string `yaml:"phone_numbers"`

```

Comment:
PhoneNumbers is not used!

Reply:
they normally come from the `message` obj

Reply:
'PhoneNumbers' is sender phone number
I will delete this config because not usable in Ghasedak API

Reply:
We call this "originator" and it's also on the Message https://github.com/messagebird/sachet/blob/master/sachet.go#L9

Reply:
in last pull request, I delete config

Reply:
in Kavenegar Provider :  Config PhoneNumbers Exist
https://github.com/messagebird/sachet/blob/master/provider/kavenegar/kavenegar.go